addons/fieldmanual/stringtable.xml

```diff
@@ -0,0 +1,488 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE_FieldManual">
+  <Package name="ACE_FieldManual">
```

Comment:
```suggestion
<Project name="ACE">
  <Package name="FieldManual">
```
Also file should be indented with 4 spaces.

Reply:
Oh this is going to be a pain. Been using Stringforge to generate the tables. What do you guys use?

Reply:
https://github.com/bux/tabler